src/thermopro_ble/device.py

```diff
@@ -0,0 +1,57 @@
+from __future__ import annotations
+
+from datetime import datetime
+from struct import pack
+from uuid import UUID
+
+from bleak import BleakClient
+from bleak.backends.device import BLEDevice
+from bleak_retry_connector import establish_connection
+
+
+class ThermoProDevice:
+    datetime_uuid = UUID("00010203-0405-0607-0809-0a0b0c0d2b11")
+
+    def __init__(self: ThermoProDevice, ble_device: BLEDevice):
+        self.ble_device = ble_device
+
+    # ----
+    # from https://github.com/koenvervloesem/bluetooth-clocks/
+    # Copyright (c) 2023 Koen Vervloesem
+    # MIT License
+    # ----
+    @staticmethod
+    def pack_datetime(dt: datetime, ampm: bool) -> bytes:
```

Comment:
```suggestion
    def pack_datetime(dt: datetime, am_pm: bool) -> bytes:
```

Please separate words with `_`